### ISSUE TITLE
instruction ?

### ISSUE THREAD

**Issue by ghost** [srinadhu/SVM#1]
please provide the proper instructions to follow the models.

**Comment by srinadhu** [comment]
Could you be more specific?

**Comment by ghost** [comment]
yeah @srinadhu, please provide the readme.md files or otherwise the instructions to follow
